File: api/java/toolgood.textfilter.api/src/main/java/toolgood/textfilter/api/Datas/KeywordTypes/KeywordtypeListResult.java
package toolgood.textfilter.api.Datas.KeywordTypes;

import java.util.List;

public class KeywordtypeListResult {
    /**
     * 返回码：0) 成功，1) 失败
     */
    public int code;
    /**
     * 返回码详情描述
     */
    public String message;

    /**
     * 敏感词类型列表
     */
    public List<KeywordtypeItem> data;
}

File: api/csharp/ToolGood.TextFilter.Api/Datas/KeywordTypes/KeywordtypeListResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace ToolGood.TextFilter.Api.Datas.KeywordTypes
{
    public class KeywordtypeListResult
    {
        /// <summary>
        /// 返回码：0) 成功，1) 失败
        /// </summary>
        [JsonProperty("code")]
        public int Code { get; set; }
        /// <summary>
        /// 返回码详情描述
        /// </summary>
        [JsonProperty("message", NullValueHandling = NullValueHandling.Ignore)]
        public string Message { get; set; }

        /// <summary>
        /// 敏感词类型列表
        /// </summary>
        [JsonProperty("data", NullValueHandling = NullValueHandling.Ignore)]
        public List<KeywordtypeItem> Data { get; set; }
    }
}
